lib/main.coffee

```diff
@@ -109,3 +111,27 @@ module.exports =
         console.warn('Copying Markdown as HTML failed', error)
       else
         atom.clipboard.write(html)
+
+  saveAsHtml: ->
+    activePane = atom.workspace.getActivePaneItem()
+    if isMarkdownPreviewView(activePane)
+      activePane.saveAs()
+      return
+
+    editor = atom.workspace.getActiveTextEditor()
+    return unless editor?
+
+    grammars = atom.config.get('markdown-preview.grammars') ? []
+    return unless editor.getGrammar().scopeName in grammars
+
+    uri = @uriForEditor(editor)
+    previewPane = atom.workspace.paneForURI(uri)
```

Comment:
I used `previewPane` to retain consistency with lines [75-77](https://github.com/atom/markdown-preview/blob/678fc98b1a0e9489914d07217175ee62e5ac7676/lib/main.coffee#L75-L77):

```coffeescript
    previewPane = atom.workspace.paneForURI(uri)
    if previewPane?
      previewPane.destroyItem(previewPane.itemForURI(uri))
``` 

Happy to amend the lines relevant to #432 to use `markdownPreviewPane` but I assume lines 75-77 should be addressed as part of a new issue.